go/framework/bridge/blackbox_test.go

```diff
@@ -0,0 +1,155 @@
+package bridge_test
```

Comment:
```suggestion
package messengerbridge_test
```

so we can have multiple bridges in the framework folder